<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>458</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>L5MapEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="separator"/>
   <addaction name="insertAction"/>
   <addaction name="deleteAction"/>
   <addaction name="separator"/>
   <addaction name="aboutAction"/>
   <addaction name="separator"/>
   <addaction name="exitAction"/>
  </widget>
  <action name="openAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/folder.png</normaloff>:/img/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="insertAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
   </property>
   <property name="text">
    <string>插入(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>插入</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/close.png</normaloff>:/img/close.png</iconset>
   </property>
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/info.png</normaloff>:/img/info.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
